modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java

```diff
@@ -411,6 +417,30 @@ private void _validateParametersUnicodeProperties(
 				_objectDefinitionPersistence.fetchByPrimaryKey(
 					objectDefinitionId);
 
+			if (parametersUnicodeProperties.containsKey(
+					"objectDefinitionExternalReferenceCode")) {
+
+				String objectDefinitionExternalReferenceCode =
+					GetterUtil.getString(
+						parametersUnicodeProperties.remove(
+							"objectDefinitionExternalReferenceCode"));
```

Comment:
This code is checking the parameter "objectDefinitionExternalReferenceCode" twice:
- parametersUnicodeProperties.containsKey()
- parametersUnicodeProperties.remove()

The method remove return NULL if it doesn't find the key. So, I believe we could refactor this code because it would only check once:
´

		String objectDefinitionExternalReferenceCode = GetterUtil.getString(
				parametersUnicodeProperties.remove("objectDefinitionExternalReferenceCode"));

		if (Validator.isNotNull(objectDefinitionExternalReferenceCode)) {}
`